core/iwasm/aot/aot_loader.c

```diff
@@ -871,6 +871,59 @@ load_name_section(const uint8 *buf, const uint8 *buf_end, AOTModule *module,
 #endif /* WASM_ENABLE_CUSTOM_NAME_SECTION != 0 */
 }
 
+#if WASM_ENABLE_STRINGREF != 0
+static bool
+load_string_literal_section(const uint8 *buf, const uint8 *buf_end,
+                            AOTModule *module, bool is_load_from_file_buf,
+                            char *error_buf, uint32 error_buf_size)
+{
+    const uint8 *p = buf, *p_end = buf_end;
+    uint32 reserved = 0, string_count = 0, i;
+    uint64 size;
+
+    read_uint32(p, p_end, reserved);
+    if (reserved != 0) {
+        set_error_buf(error_buf, error_buf_size,
+                      "invalid reserved slot in string literal count");
+        goto fail;
+    }
+
+    read_uint32(p, p_end, string_count);
+    if (string_count == 0) {
+        set_error_buf(error_buf, error_buf_size,
+                      "invalid string literal count");
+        goto fail;
+    }
+    module->string_literal_count = string_count;
+
+    size = sizeof(char *) * string_count;
```

Comment:
Had better `(uint64)sizeof(char *)` to avoid possible integer overflow.

Reply:
Done, thanks